### PR TITLE
Change links to Monitoring docs for new route

### PR DESCRIPTION
**Pull request by meker12**

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -360,17 +360,17 @@
...
                        <div class="card-body">
                            <div class="list">
                                <div class="list-column">
                                    <ul>
-                                       <li><a href="/docs/cloud-monitoring/getting-started/">Quick Start</a></li>
-                                       <li><a href="/docs/cloud-monitoring/v1/developer-guide/#document-api-reference">API Reference</a></li>
-                                       <li><a href="/docs/cloud-monitoring/v1/developer-guide/#release-notes">Release Notes</a></li>
-                                       <li><a href="/docs/cloud-monitoring/v1/developer-guide/">Developer Guide</a></li>
-                                       <li><a href="/docs/cloud-monitoring/v1/developer-guide/#document-tech-ref">Technical Reference</a></li>
+                                       <li><a href="/docs/rackspace-monitoring/getting-started/">Quick Start</a></li>
```
Comment (smashwilson) [inline index.rst]: Is `/docs/rackspace-monitoring/getting-started/` mapped anywhere? It doesn't seem to be, only `/docs/rackspace-monitoring/v1/developer-guide/`.

Comment (smashwilson) [inline index.rst]: Oh, I see, that was coming from [this directory in docs-quickstart](https://github.com/rackerlabs/docs-quickstart/tree/master/cloud-monitoring/getting-started). I'm guessing that you'll want to rename and redirect that, too.

**Comment by smashwilson** [comment]
Did you assign me to this because you want me to review this, or merge it, or... ?

**Comment by meker12** [comment]
Assigned so you could merge, and thanks for noticing the Getting Started link issue

**Pull request closed without merging**